app/app.rb

```diff
@@ -60,5 +60,9 @@ class App < Padrino::Application
     #     render 'errors/500'
     #   end
     #
+
+    use OmniAuth::Builder do
+      provider :facebook, '1354728827916287','4412667c2bed46b3ed06d8989305c766'
```

Comment:
@wikimatze yes i made a mistake and realized it after pushing it. I will use dotenv for these kind of stuff. I feel so embarassed